plugin/trino-hudi/src/test/resources/hudi-testing-data/stock_ticks_mor_exception_test/.hoodie/20211221030120532.deltacommit

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
add a readme file describing how the table was created

Reply:
Done